### ISSUE TITLE
  It is not displayed in my Magento's backend order grid

### ISSUE THREAD

**Issue by dean224** [gsafcik/OrdersPlus#2]
magento ce 1.9.3.1
I have cleared the cache and there is no display in order gird.

**Comment by gsafcik** [comment]
@dean224 Are you using flat tables by chance (I haven't tested with flat tables)? What cache options are you using? I have CE 1.9.3.1 installed and will test to confirm that it works as expected. In the meantime, please give me more details. Are using another extension that overrides the sales order grid page as well? Perhaps a screenshot?

**Comment by dean224** [comment]
@gsafcik Yes. I have enabled flat tabel, 
this is a oreder plus,https://www.atwix.com/magento/column-to-orders-grid/ 
which has been installed before, and has now been uninstalled

**Comment by gsafcik** [comment]
@dean224 Atwix's extension would've competed with this one. Please make sure that it is fully uninstalled. Now, I have not tested this with flat tables enabled. Can you do two things to test for me (only if you have a dev environment - do not test on production)?

1. If you reindex, does the data you want show?
2. If you disable flat tables, does it work as expected?

Of course, I will not ask you to do this stuff to make it work in production. However, I'd like some information so that I can update this module accordingly.

**Comment by dean224** [comment]
@gsafcik Thank you for your reply.
I have disable flat tables,and reindexed,but ,the data still no show.
But I tested a  new Magento system and turned on flat table, which worked fine.
Maybe my other plugins are conflicting.

**Comment by gsafcik** [comment]
@dean224 That is good news about it working on a fresh install (because I was about to tell you the same thing). Would you be willing to list out the extensions you're using so that I can see if anything stands out right away?

**Comment by dean224** [comment]
@gsafcik Thank you for your reply.
My screenshot lists the plugins I use
You can see the screenshot here
[https://ibb.co/iHnDVF](https://ibb.co/iHnDVF)

**Comment by gsafcik** [comment]
@dean224 From the provided screenshot, my guess would be that Amasty or Raveinfosys are conflicting. I would try disabling those first and see if it works...

**Comment by gsafcik** [comment]
@dean224 any progress?

**Comment by dean224** [comment]
@gsafcik ,yes,you are right.when I disable Raveinfosys Plug-in,the order grid is blank!But when I uninstall it,I get a error:
Fatal error: Call to a member function setSaveParametersInSession() on boolean in D:\phpStudy\WWW\magento\dev1.cc\app\code\core\Mage\Adminhtml\Block\Widget\Grid\Container.php on line 66
